pyproject.toml

```
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "zcash-core-devtools"
+version = "0.1.0"
+description = "Zcash Core Developer Tools"
```

There is no "Zcash Core"; there's the ECC Core Team, but that's handled within the DAG script.
```suggestion
name = "zcash-developer-tools"
version = "0.1.0"
description = "Zcash Developer Tools"
```